Samples/auth/Office-Add-in-Microsoft-Graph-React/README.md

```diff
@@ -0,0 +1,106 @@
+
+* A code editor.
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Office Add-in Microsoft Graph React | Microsoft
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0  | August 29th, 2019| Initial release
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+----------
+
+## Build and run the solution
+
+### Configure the solution
+
+1. Register your application using the [Azure Management Portal](https://manage.windowsazure.com). **Log in with the identity of an administrator of your Office 365 tenancy to ensure that you are working in an Azure Active Directory that is associated with that tenancy.** To learn how to register your application, see [Register an application with the Microsoft Identity Platform](https://docs.microsoft.com/graph/auth-register-app-v2). Use the following settings:
+
+ - REDIRCT URI: `https://localhost:3000/login/login.html`
+ - SUPPORTED ACCOUNT TYPES: "**Accounts in any organizational directory**"
+ - IMPLICIT GRANT: Enable *both* options: **Access tokens** and **ID tokens**
+ - API PERMISSIONS: none. Do not request any. 
+ - SECRETS: none. The sample uses the OAuth 2.0 Implicit Flow, which requires no secrets.
+
+	> Note: After you register your application, copy the **Application (client) ID** on the **Overview** blade of the App Registration in the Azure Management Portal. 
+	 
+3.  In the code editor, open the `/login/login.ts` file in the project. Near the top is a configuration property called `clientId`. Replace the placeholder value with the application ID you copied from the registration. Save and close the file.
+
+4. Follow the instructions at [SSL](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md#method-2-manually-install-the-certificate) to trust a certificate. Use method 2.
+
+5. Open a **Command Prompt** *as an administrator*. Run the command `npm install`.
+
+### Run the solution
+
+1. In the command prompt, run the command `start npm start`. This will open a second command prompt, build the project and then start a server (with dev mode settings). It takes from 5 to 30 seconds. When it finishes, the last line should say `Compiled successfully`. Minimize this command prompt.
+2. Back in the original command prompt, run the command `npm run sideload`. This will launch Excel and install the add-in in it. After a few seconds, a **OneDrive Files** group appears on the right end of the **Home** ribbon with a button named **Open Add-in**.
+3. Click the **Open Add-in** to open the task pane add-in.
+4. The pages and buttons in the add-in are self-explanatory. 
+
+	> Note: The first time that you press the **Connect to Office 365** button and sign in, you will be prompted to consent to the add-in. 
```

Comment:
Unfortunately the login dialog is blank. I get a 404 error that it cannot find the login.js file. In my compiled code the login.js has a completely different name like login.4582739.js or something. For some reason the build is not dropping or naming the files as expected by the solution.